plugins/scaffolder/report.api.md

```diff
@@ -618,6 +618,7 @@ export const scaffolderPlugin: BackstagePlugin<
     editor: SubRouteRef<undefined>;
     customFields: SubRouteRef<undefined>;
     templateForm: SubRouteRef<undefined>;
+    templateExtensions: SubRouteRef<undefined>;
```

Comment:
```suggestion
    templatingExtensions: SubRouteRef<undefined>;
```

Reply:
Sure, I'm not picky about that. I can't remember offhand but I'm guessing we may already have used that terminology on the backend. If so do we want to go ahead and change those as well since they should have been alpha?

Reply:
Yeah if there's anything that stands out as sounds better with `templating` instead then lets go ahead and update that. `minor` is fine for making breaking changes to `/alpha` export also.

i didn't see so much earlier when I searched so maybe there's not too much.